schema/helpers/transform.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Transform",
+    "description": "Layer transform",
+    "allOf": [
+        {
+            "properties": {
+                "a": {
+                    "title": "Anchor Point",
+                    "description": "Anchor point: a position (relative to its parent) around which transformations are applied (ie: center for rotation / scale)",
+                    "$ref": "#/$defs/properties/position"
+                },
+                "p": {
+                    "title": "Position",
+                    "description": "Position / Translation",
+                    "$ref": "#/$defs/properties/position"
+                },
+                "r": {
+                    "title": "Rotation",
+                    "description": "Rotation in degrees, clockwise",
+                    "$ref": "#/$defs/properties/value"
+                },
+                "s": {
+                    "title": "Scale",
+                    "description": "Scale factor, `[100, 100]` for no scaling",
+                    "$ref": "#/$defs/properties/multi-dimensional"
+                },
+                "o": {
+                    "title": "Opacity",
+                    "$ref": "#/$defs/properties/value"
+                },
+                "sk": {
```

Comment:
should we specify that skew and skew axis are not supported in layers and that they are only supported as shape transforms?

Reply:
I think some players support them in layers as well, perhaps this can be specified in the profile definitions